code/modules/clothing/suits/armor.dm

```diff
@@ -74,6 +74,90 @@
 		usr.update_action_buttons()
 		return 1
 
+/obj/item/clothing/suit/armor/vest/breach1
+	name = "chemical corps vest"
+	desc = "A reinforced combat vest from solfed surplus stockpiles. Originally an enviroment protection suit clad in armor plating for full body protection and points for mounting specialist equipment. This set seems void of typical additions."
+	icon_state = "breach1"
+	item_state = "breach1"
+	blood_overlay_type = "armor"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
+	stiffness = LIGHT_STIFFNESS
+	price_tag = 600
```

Comment:
Kinda low given that it's meant to be a mid tier upgrade over other generalized armors but lower than assault armor. Especially since it has 0 slowdown.